osutil/user/getent.go

```diff
@@ -24,13 +24,42 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"os"
 	"os/exec"
+	"path/filepath"
 	"strconv"
 	"strings"
 )
 
+const (
+	defaultGetentSearchPath = "/usr/bin:/bin:/usr/sbin:/sbin"
```

Comment:
For completeness sake, we may just as well mirror: https://github.com/canonical/snapd/blob/master/cmd/snap-confine/snap-confine.c#L887 minus *games.

Reply:
@bboozzoo any strong reason to or not to include the local variants? admittedly they by definition don't contain distro binaires

Reply:
Updated, it's now using the same paths as the apparmor binary lookup